pkg/resources/secret_common.go

```diff
@@ -33,6 +33,11 @@ var secretCommonSchema = map[string]*schema.Schema{
 		ForceNew:         true,
 		DiffSuppressFunc: suppressIdentifierQuoting,
 	},
+	"secret_type": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Specifies a type for the secret.",
```

Comment:
You can add a comment saying that we use this field for checking external values and recreating the resources, if needed. The reason is that in general we have our computed fields in `show_output` and `describe_output` and I think users expect configurable fields here.

Reply:
will be added